packages/typescript-config-react/tsconfig.json

```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "https://json.schemastore.org/tsconfig",
```

Comment:
question: is this supposed to be run a nextjs app? I wonder, in case, if we just want to use the tsconfig generatd by `yarn dlx create-next-app@latest` command and commit the whole skeleton

Reply:
yep, that sounds good.
I created the scaffold using the `create-next-app` package and committed the whole skeleton (I removed some noisy files).

Moreover, I extracted the common configuration of the `tsconfig` file and put it into the `typescript-config-react` package.
The webapp, then, imports that configuration and can add some other options.